blocks/README.md

```diff
@@ -40,3 +40,27 @@ blocks/
 ```
 
 Please refer to the official [BEM](https://en.bem.info/methodology/quick-start/#introduction) or [BEM Nested](https://en.bem.info/methodology/quick-start/#file-structure) documentation for details if needed.
+
+### Using media queries
+
+For all components, we use the Desktop-first design approach, programming them initially for the desktop version, and subsequently adapting them to breakpoints: `1199px, 659px,` using `max-width` for media queries
+
+Example:
+
+```bash
```

Comment:
@NazarPauk Let's make a minor formatting adjustment to make the code example in the documentation more precise and consistent.

Please replace 'bash' with 'CSS' after the triple backticks (```) to indicate that the code block contains CSS styling.  Also, let's use `/* */` for comments within the CSS code block.

This adjustment will bring some familiar CSS styling to our code example, making it easier to read and work with.

Reply:
Done✅